internal/pkg/template/templates/workloads/services/static-site/manifest.yml

```diff
@@ -16,19 +16,10 @@ files:
 {{- else}}
 files:
 {{- range $file := .FileUploads}}
-  - source: {{$.Source}}
-    destination: {{$.Destination}}
-    {{- if $.Context}}
-    context: {{$.Context}}
-    {{- end}}
-    {{- if $.Recursive}}
-    recursive: {{$.Recursive}}
-    {{- end}}
-    {{- if $.Reinclude.StringSlice}}
-    reinclude: {{$.Reinclude.StringSlice}}
-    {{- end}}
-    {{- if $.Exclude.StringSlice}}
-    exclude: {{$.Exclude.StringSlice}}
+  - source: {{$file.Source}}
+    {{- if $file.Recursive}}
+    destination: assets
```

Comment:
why `assets`..? isn't the bucket an asset bucket already?

Reply:
changed per offline conversation